2017/JULY/JULY17/EXPTREE/EXPTREE.py

```diff
@@ -0,0 +1,57 @@
+#__BY:sirjan13__
+
+
+def gcd(n,m):
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpgf_vkzf9/2017/JULY/JULY17/EXPTREE/EXPTREE.py
+++ b/tmp/tmpgf_vkzf9/2017/JULY/JULY17/EXPTREE/EXPTREE.py
@@ -1,7 +1,7 @@
 #__BY:sirjan13__
 
 
-def gcd(n,m):
+def gcd(n, m):
     if n == 0:
         return m
     return gcd(m % n, n)
```